src/components/AccountsManagementComponent.jsx

```diff
@@ -159,140 +190,341 @@ class AccountsManagementComponent extends Component {
+                        )}
+                        <Typography variant='caption'>
+                          $ {this.toCurrencyLocaleString(account.totalMarketValue)}
+                        </Typography>
+                      </Box>
+                      <Box ml={1}>
+                        {expanded ? (
+                          <ExpandLessIcon className={classes.iconBtn} />
+                        ) : (
+                          <ExpandMoreIcon className={classes.iconBtn} />
+                        )}
+                      </Box>
+                    </Box>
+                  </div>
+                </Box>
+              </ListItem>
+              {expanded && (
+                <Box
+                  display='flex'
+                  flexDirection='column'
+                  alignItems='stretch'
+                  justifyContent='flex-start'
+                  pl={4}
+                  pr={3}
+                >
+                  <ListItem style={{ padding: 0 }}>
+                    {this.rednerAccountActionButtons(account)}
+                  </ListItem>
+                  {account.assets.length > 1 &&
+                    account.assets.map(accountData => {
+                      return (
+                        <>
+                          <Divider style={{ marginLeft: 10 }} />
+                          <ListItem className={classes.listItemBase}>
+                            <Box
+                              display='flex'
+                              flexDirection='row'
+                              justifyContent='space-between'
+                              style={{ width: '100%' }}
+                            >
+                              <Typography variant='body2'>
+                                {`${getCryptoSymbol(accountData.cryptoType)}`}{' '}
+                                {`${this.toCurrencyLocaleString(
+                                  accountData.balanceInStandardUnit
+                                )}`}
+                              </Typography>
+                              <Typography variant='body2'>
+                                $ {this.toCurrencyLocaleString(accountData.marketValue)}
+                              </Typography>
+                            </Box>
+                          </ListItem>
+                        </>
+                      )
+                    })}
+                </Box>
+              )}
+            </React.Fragment>
+          )
+        })}
+        <Divider />
+      </List>
+    )
+  }
+
   render () {
-    const { classes, online } = this.props
-    const { addAccountModal } = this.state
+    const { classes, categorizedAccounts, actionsPending, online } = this.props
+    const {
+      addAccountModal,
+      deleteConfirmModal,
+      changeNameModal,
+      windowWidth,
+      addressQRCodeDialog,
+      chosenAccount
+    } = this.state
+
+    const wide = windowWidth >= 800
```

Comment:
somewhat adhoc

Reply:
use https://v4-0-2.material-ui.com/customization/breakpoints/#withwidth-options-higher-order-component
and breakpoint sm

Reply:
This option was considered, but not used. With our design, the window with for the wide table cannot be lower than 800px. In some cases, X-overflow will occur. And 800px is neither sm(600px) nor md(960px). If we were to use breakpoint here, we need to change our definition for breakpoint, but that might cause unwanted side-effect on other components, especially  those use <Grid>

Reply:
i c